canonical/vanilla-framework
# Update rule-highlight pattern to use to appropriate class

## Done

Update rule-highlight pattern to use to appropriate class

## QA

- Open [demo](https://vanilla-framework-4759.demos.haus/docs/patterns/rule)
- Make sure example shows highlight rules

### Check if PR is ready for release

If this PR contains Vanilla SCSS code changes, it should contain the following changes to make sure it's ready for the release:

  - `Feature 🎁`, `Breaking Change 💣`, `Bug 🐛`, `Documentation 📝`, `Maintenance 🔨`.


## Screenshots
Before:
![image](https://user-images.githubusercontent.com/58276363/236160581-d02ae316-66a5-418d-b334-096ba239c356.png)

After:

<img width="1295" alt="image" src="https://user-images.githubusercontent.com/83575/236170133-53d830bb-db7d-4173-b444-f05b5185e780.png">
